### PR TITLE
New article Integrating new payment provider

### PR DESCRIPTION
**Pull request by ricardoaerobr**
#### Types of changes
- [X] New content (guides, endpoints, app documentation)
- [ ] Improvement (make a documentation even better)
- [ ] Fix (fix a documentation error)
- [ ] Spelling and grammar accuracy (self-explanatory)

PR created to publish the new article: ​​Integrating a new payment provider on VTEX (EDU-9951).
Note: Content previously reviewed by Bruno and translated by the localization team.

### CODE REVIEW

File: docs/guides/Integration-Guides/payments-integration-guide/integrating-a-new-payment-provider-on-vtex.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
+---
+title: "Integrating a new payment provider on VTEX"
+slug: "integrating-a-new-payment-provider-on-vtex"
+hidden: false
+createdAt: "2023-08-25T00:00:15.623Z"
+updatedAt: ""
+---
+This article is divided into stages that explain the integration flow for a new payment provider on VTEX. Before completing each stage and moving on, make sure to check and understand the information in the related links, as well as any prerequisites and settings that need to be in place in the current stage.
```
Comment (karenkrieger) [inline docs/guides/Integration-Guides/payments-integration-guide/integrating-a-new-payment-provider-on-vtex.md:8]: Em vez de `stages` acho que `steps` ou `sections` faz mais sentido

File: docs/guides/Integration-Guides/payments-integration-guide/integrating-a-new-payment-provider-on-vtex.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+- [VTEX Sales App - Payments](https://help.vtex.com/en/tracks/instore-pagamentos--43B4Nr7uZva5UdwWEt3PEy)
+- [Payment Provider Protocol applied to payments with POS](https://developers.vtex.com/docs/guides/payments-integration-ppp-applied-to-pos)
+
+### Operating payments in both physical stores and ecommerce website
+
+If you want to offer a payment experience in physical stores and on the ecommerce website, you will need to configure settings for both cases.
+
+To learn more about this type of operation, in addition to the documentation references linked in previous sections, see [Unified Commerce Strategies](https://help.vtex.com/en/tracks/estrategias-de-comercio-unificado--3WGDRRhc3vf1MJb9zGncnv/2LGAiUnHES1enjHsfi8fI3).
+
+> ℹ️ You can develop a single connector configured to work in both environments (physical stores and ecommerce website). You don't have to create a connector for each type of operation.
```
Comment (karenkrieger) [inline docs/guides/Integration-Guides/payments-integration-guide/integrating-a-new-payment-provider-on-vtex.md:68]:
```suggestion
> ℹ️ You can develop a single connector configured to work in both environments (physical stores and ecommerce websites). You do not have to create a connector for each type of operation.
```

File: docs/guides/Integration-Guides/payments-integration-guide/integrating-a-new-payment-provider-on-vtex.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+To understand how the Payment Provider Protocol works and the routes used in the payment and authentication flows, see:
+
+- [Payment Provider Protocol - Overview](https://help.vtex.com/en/tutorial/payment-provider-protocol--RdsT2spdq80MMwwOeEq0m)
+- [Payment Provider Protocol - API References](https://developers.vtex.com/docs/api-reference/payment-provider-protocol)
+
+After making sure you meet the PPP implementation [prerequisites](https://help.vtex.com/en/tutorial/payment-provider-protocol--RdsT2spdq80MMwwOeEq0m#implementation-prerequisites), the next step is defining how your connector will work in the VTEX environment.
+
+## 2. Defining the operation environment
+
+On VTEX, a payment connector can operate in three different environments: ecommerce websites, physical stores, or both. Select the option that matches the operating characteristics of your connector.
```
Comment (karenkrieger) [inline docs/guides/Integration-Guides/payments-integration-guide/integrating-a-new-payment-provider-on-vtex.md:41]:
```suggestion
On VTEX, a payment connector can operate in three different contexts: ecommerce websites, physical stores, or both. Select the option that matches the operating characteristics of your connector.
```

File: docs/guides/Integration-Guides/payments-integration-guide/integrating-a-new-payment-provider-on-vtex.md [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+
+### Developing with VTEX infrastructure (VTEX IO)
+
+If you want to develop your connector using VTEX infrastructure, you can use our [VTEX IO](https://developers.vtex.com/docs/guides/vtex-io-documentation-what-is-vtex-io), our proprietary development platform. VTEX IO offers the following advantages:
+
+- Greater integration agility by reusing VTEX templates and clients.
+- [Serverless](https://www.redhat.com/en/topics/cloud-native-apps/what-is-serverless) environment with no need to invest additional financial resources related to infrastructure, reliability, and security management.
+- Simplified authentication, authorization, and configuration resources that are integrated with all other VTEX systems.
+
+If you use our infrastructure, you can also use our VTEX IO boilerplate, __Payment Provider Framework (PPF)__, which makes creating the connector faster in comparison to using your own infrastructure. With PPF, you don't have to manually configure the API routes, response and request body default formats, and sensitive information transfer (such as credit card information).
```
Comment (karenkrieger) [inline docs/guides/Integration-Guides/payments-integration-guide/integrating-a-new-payment-provider-on-vtex.md:94]:
```suggestion
If you use our infrastructure, you can also use our VTEX IO boilerplate, __Payment Provider Framework (PPF)__, which makes creating the connector faster in comparison to using your own infrastructure. With PPF, you do not have to manually configure the API routes, response and request body default formats, and sensitive information transfer (such as credit card information).
```

File: docs/guides/Integration-Guides/payments-integration-guide/integrating-a-new-payment-provider-on-vtex.md [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+
+### Transparent
+
+In the transparent purchase flow, the customer enters the payment information and places the order directly in the checkout of the VTEX store. This is the most widely used flow type because it allows completing all payment steps in the same environment, which improves the customer purchase experience.
+
+To learn more about the transparent purchase flow, see [What is transparent checkout?](https://help.vtex.com/en/tutorial/what-is-transparent-checkout--2Y4ECegUmcYUggmck2GOwe).
+
+### Custom
+
+If you don't want to use the native VTEX purchase flow (transparent), you have two options to create custom payment experiences for your customer: Payment App and Redirect.
```
Comment (karenkrieger) [inline docs/guides/Integration-Guides/payments-integration-guide/integrating-a-new-payment-provider-on-vtex.md:114]:
```suggestion
If you do not want to use the native VTEX purchase flow (transparent), you have two options to create custom payment experiences for your customer: Payment App and Redirect.
```

**Review by karenkrieger — CHANGES_REQUESTED** [review]

File: docs/guides/Integration-Guides/payments-integration-guide/integrating-a-new-payment-provider-on-vtex.md (hunk 1 above)
Comment (ricardoaerobr) [reply docs/guides/Integration-Guides/payments-integration-guide/integrating-a-new-payment-provider-on-vtex.md]: Boa! Corrigido.

**Review by karenkrieger — APPROVED** [review]
LGTM

**Pull request merged**